ERCS/erc-7562.md

```diff
@@ -246,6 +246,11 @@ The following reputation rules apply for all staked entities, and for unstaked p
 
 ### Entity-specific Rules
 
+definition: 
+    **accountable**:an entity that will be banned if the UserOperation revert when creating a bundle (GREP-040)
```

Comment:
"Accountable" only applies to banning, and not throttling cases?  According to this definition a paymaster that returns "valid" for two UserOps separately and then causes a bundle revert, it is "accountable" but a paymaster that just causes a UserOp (or any number of UserOps) to become invalid separately after being valid in 1st simulation (e.g. by flipping a flag in its own storage) is not "accountable".  Is that the intention here?

Reply:
there are 2 separate mechanisms
1. [opsSeen/opsIncluded](https://eips.ethereum.org/EIPS/eip-7562#reputation-definitions) on every entity in the userop, that collects "reputation" over time. over a long time, too many opsSeen (w/o opsIncluded, for whatever reason) eventually throttle and ban the entity.
1a. special case for PM [EREP-015]: its opsSeen is not included for any 2nd validation failure caused by account/factory
2. [GREP-040] Direct ban, by failing bundle creation (which sets opsSeen=10000, opsIncluded = 0 - a ban for few days)

the question: are there any other case?

Reply:
I think these are currently the only cases.  My question was about the "accountable" definition.  Is an entity accountable for any invalidation it causes, or only for the 2nd case?

Reply:
- the original idea was to manage opsSeen/opsIncluded separated for each entity, regardless of who caused the error (as in some cases, we couldn't really tell)
- then we added a specific rule (EREP-015) for paymaster
- maybe we should try to do it for other entities too. That is: with "accountable" entity, we "undo" the increment of other entities.
the flow:
- sendUserOp: increase opsSeen count for sender, paymaster, factory
- 2nd validation: depending on staked entity and error cause, undo some opsSeen (need to create a table...)
- createBundle: if crashes, perform the opsSeen undo above, but also ban the offending entity.

Reply:
@yoavw , @shahafn , @forshtat  - comments?